Grant Application - Reference blockchain system.md

```diff
@@ -0,0 +1,286 @@
+
+In our system, everyone can have equal opportunity to participate in and contribute to the development and management of online service by freely changing and improving it with a new version, and directly putting the new work into the market for users to choose. In this way the collaboration and production within the organization are fully open and equal to everyone, and the development and governance of the online service are fully decentralized and autonomous. 
+
+![img](https://github.com/hoho-yin/Grants-Program/blob/hoho-yin-patch-1/DAO.png)
+
+**For users and service providers** (like drivers in Uber, or bloggers in YouTube):
+
+As our system provides unlimited different versions for each online service, users can freely choose what they want even if it’s not the choice of majority without compromise. As different versions are interconnected sharing the same network and resources without monopoly and isolation, users can freely switch among them when having better choice without any cost or barrier. 
+
+In this way, users can get the best online services to fully protect their individual digital rights, freedom and properties. As online services are fully decentralized made by the public and for the public, they can be truly reliable and regulated without having privileges or monopoly to suppress users and service providers for self-interests, while still fully incentivizing to maintain and improve their online services continuously. 
+
+![img](https://github.com/hoho-yin/Grants-Program/blob/hoho-yin-patch-1/users.png)
+
+
+**Use case**
+
+For example, currently one DeFi application couldn’t have multiple versions made by different people. If others change a DeFi protocol to make a new version of it, it couldn’t get the internal network and resources of the original version. For instance, Sushiswap improved Uniswap, but Uniswap couldn’t open and share its liquidity pool with Sushiswap. Sushiswap has to build a new protocol completely independent from Uniswap even though it’s based on Uniswap. In order to have its own liquidity, Sushiswap launched vampire attacks to scramble for Uniswap’s liquidity. 
+
+With our reference system, Uniswap can have unlimited new versions just like branches or forks made by anyone including Sushiswap. Uniswap opens and shares its internal users and network and allows anyone to make any changes on itself, so Sushiswap can access the liquidity pools of Uniswap and build their protocol based on them directly with no need to build their own from scratch. In return, the earnings of the Sushiswap will be distributed with Uniswap as the incentive for its opening and sharing. Users don’t have to choose between Uniswap and Sushiswap, but can use both at the same time without separation.
```

Comment:
We think Web3 services should be fully decentralized. The development of each service should be out of owner's control. Owner should not control and determine the future of the service like roadmap following his intention, but should allow different people can improve and change it freely as long as markets and users need, even beyond owner's scope. 

For example although Uniswap doesn't intend to have governance token in the beginning, it should still allow others like Sushiswap to add platform token in the new version developed by Sushiswap. Users can freely choose old or new version. Old and new version can share the same liquidity pool, so Sushiswap can directly build their project based on Uniswap rather than having to build a new project from scratch and launching vampire attack to scramble the liquidity of Uniswap. Such collaboration rather than vicious competition would be beneficial to both Uniswap and Sushiswap.